.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "libraries/fc"]
```

Comment:
should we vendor these in instead of using submodules?  especially considering that we are now pulling in an external one?